.editorconfig

```diff
@@ -0,0 +1,20 @@
+# https://editorconfig.org/
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+continuation_indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Isn't this a disaster for Markdown? Stupidly, its convention for a line-break is two space characters at the end of the line before the break.

Reply:
No. You and i explored this a few years ago:
https://github.com/folio-org/Net-Z3950-FOLIO/pull/22#issuecomment-974590408

The outcome was the solution in the FAQ that i referred to in this pull-request\
https://dev.folio.org/faqs/how-to-use-editorconfig/#markdown-line-break

That previous sentence has an example as backslash terminator ... `pull-request\`

Reply:
Argh! My memory!

I was amused to find in the original discussion:

> Excellent! Then that is what I will use going forward (assuming I remember).

:-)